app/scripts/lib/errors.js

```diff
@@ -136,6 +136,16 @@ define(function (require, exports, module) {
       return error.errno === code;
     },
 
+    /**
+     * Check if an error was created by this module.
+     *
+     * @param {object} error - error to check
+     * @returns {boolean} - true if from this module, false otw.
+     */
+    createdError: function (error) {
```

Comment:
There's an `is` directly above this function. I was struggling for a name, what I'm trying to get at is "did this module create this error?" I tossed around `hasError`, `ownsError`, `isErrorFrom`, `isErrorCreatedBy`... I'm not happy with the current name either, any other suggestions?